mifospay/src/main/java/org/mifos/mobilewallet/mifospay/editprofile/ui/EditProfileActivity.java

```diff
@@ -161,7 +163,8 @@ public void onChangePasswordClicked() {
 
     @OnClick(R.id.btn_change_passcode)
     public void onChangePasscodeClicked() {
-        // TODO: it's not supported by the api???
+        startActivity(new Intent(getApplicationContext(), PassCodeActivity.class));
```

Comment:
@Bhavnaharitsa why have you added this?
Also, please fix the Travis build.

Reply:
I'm sorry I guess this pr has got wrong changes